src/components/Components/ComponentsListView/ComponentsListRow.tsx

```diff
@@ -30,14 +27,12 @@ export const getContainerImageLink = (url: string) => {
 
 const ComponentsListRow: React.FC<
   RowFunctionArgs<ComponentWithLatestBuildPipeline, PacStatesForComponents>
-> = ({ obj: component, customData }) => {
+> = ({ obj: component }) => {
```

Comment:
If we do not need that `customData`, Please remove the data passed to customData in the ComponentListView Table component.

Reply:
Great catch! Removed: https://github.com/konflux-ci/konflux-ui/pull/24/files#diff-e0b2228fc4bc131576cdbbe79d50a98ad325b7a7d69d746f3ae5a4a2caf6dfabL255